clang/bindings/python/tests/cindex/test_cdb.py

```diff
@@ -10,7 +10,7 @@
 import sys
 from pathlib import Path
 
-kInputsDir = os.path.join(os.path.dirname(__file__), "INPUTS")
+k_inputs_dir = os.path.join(os.path.dirname(__file__), "INPUTS")
```

Comment:
Can you also drop `k_` prefix? According to PEP 8, naming convention for global variables is the same as for functions, and we don't prefix functions with `k_`.

Reply:
Sounds good. I've been wondering the entire time what the `k` us supposed to stand for.

Reply:
"Konstant", I guess